data/human/hails.txt

```diff
@@ -742,10 +784,140 @@ phrase "friendly civilian"
 		"fly a multipurpose ship that has room for cargo, shields, and heavy weapons"
 		"equip anti-missile turrets so they can't get you with missiles, and then you can run away"
 		"have a fleet with interceptors to tie them up while the cargo ships can warp to safety"
+		"slap on a scram drive and easily jump away"
+		"overheat them with a plasma volley"
+		"leave yourself unarmed, so they don't prioritise you"
+		"use an array of proton turrets, to attack from afar"
+		"use bigger cargo ships, so your fleet is harder to destroy"
+		"avoid taking jobs near the frontier"
+		"not to use so many cargo ships that you become irresistable to them"
 	word
 		"."
 		"!"
 
+
+phrase "friendly civilian"
+	word
+		"You won't believe"
+		"You couldn't imagine"
+		"It's absurd"
+		"It's ridiculous"
+		"It's bizzare"
+	word
+		" how much "
+	word
+		"people"
+		"rich people"
+		"retirees"
+		"artists"
+		"celebrities"
+		"politicians"
+		"nerds"
+		"programmers"
+		"scientists"
+		"tourists"
+	word
+		" are willing to pay for"
+	word
+		" "
```

Comment:
Why have this extra word that's just a space if `" are willing to pay for"` already has a space at the beginning?